src/mavsdk/core/mavlink_mission_transfer_server.cpp

```diff
@@ -56,6 +56,39 @@ MavlinkMissionTransferServer::receive_incoming_items_async(
     return std::weak_ptr<WorkItem>(ptr);
 }
 
+std::weak_ptr<MavlinkMissionTransferServer::WorkItem>
+MavlinkMissionTransferServer::send_outgoing_items_async(
+    uint8_t type,
+    const std::vector<ItemInt>& items,
+    uint8_t target_system,
+    uint8_t target_component,
+    ResultCallback callback)
+{
+    if (!_int_messages_supported) {
+        if (callback) {
+            LogErr() << "Int messages are not supported.";
+            callback(Result::IntMessagesNotSupported);
+        }
+        return {};
```

Comment:
It's a good point, admittedly I copied this from `MavlinkMissionTransferClient::upload_items_async()` and scanned over it but didn't think about this super carefully. Reading through the server code `_int_messages_supported` is initialized to `true`, and never changed. the `set_int_messages_supported()` method has no actual implementation in `MavlinkMissionTransferServer`, which means nobody is calling it, and all of it can be removed. I think I would propose removing this check along with all references to `_int_message_supported` in this class. Thoughts?